src/script/standard.h

```diff
@@ -89,4 +90,10 @@ CScript GetScriptForMultisig(int nRequired, const std::vector<CPubKey>& keys);
 CScript GetScriptForStakeDelegation(const CKeyID& stakingKey, const CKeyID& spendingKey);
 CScript GetScriptForOpReturn(const uint256& message);
 
+namespace Standard {
+
+std::string EncodeDestination(const CWDestination &address, const CChainParams::Base58Type addrType = CChainParams::PUBKEY_ADDRESS);
+
+} // End Standard namespace
```

Comment:
I don't think that we should put in `standard/script` functions that deal with IO (converting between scripts/addresses and strings). We should put them in a separate file (including `key_io_sapling.h` and `key_io.h`) outside of the script folder.

Reply:
Oh.. I see it's cleaned up later in 95a58e4a975699ad6f9883e832cba027d7f0bd54
All good then.
We can remove the inclusion of `sapling/key_io_sapling.h` too.

Reply:
yeah, can remove that include from 95a58e4 now.

Reply:
even the base58 include is redundant as well.